Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -225,33 +234,88 @@ def _join_state_geo_df(data: pd.DataFrame,
 def _join_county_geo_df(data: pd.DataFrame,
                         county_col: str,
                         geo_info: gpd.GeoDataFrame,
-                        join_type: str = "right") -> gpd.GeoDataFrame:
+                        join_type: str = "right",
+                        combine_megacounties: bool = False) -> gpd.GeoDataFrame:
     """Join DF information to polygon information in a GeoDF at the county level.
 
     Counties with no direct key in the data DF will have the megacounty value joined.
 
     :param data: DF with county info.
     :param county_col: name of column in `data` containing county info to join on.
     :param geo_info: GeoDF of county shape info read from Census shapefiles.
+    :param join_type: Type of join to do between input data (left side) and geo data (right side).
```

Comment:
What are the possible options?

Reply:
Added - left, right, inner, outer. I didn't add an explanation for what each of those is for brevity, but could if we tihnk it's important. Same for comments below.